scene/gui/text_edit.cpp

```diff
@@ -681,6 +681,17 @@ void TextEdit::_notification(int p_what) {
 				}
 			}
 
+			if (line_length_guideline) {
+				int x = xmargin_beg + cache.font->get_char_size('0').width * line_length_guideline_col - cursor.x_ofs;
+				if (x > xmargin_beg && x < xmargin_end) {
+#ifdef TOOLS_ENABLED
```

Comment:
As mentioned in #29730, we should stop doing it this way, but use proper theme constants.

Reply:
As discussed on IRC, the same is already done in various places in this class, so we can merge this new one and fix all later.